app/controllers/admins/company_values_controller.rb

```diff
@@ -0,0 +1,62 @@
+module Admins
+  class CompanyValuesController < AdminsController
+    def index
+      @company_values = CompanyValue.all
+    end
+
+    def show
+      @company_value = CompanyValue.find(params[:id])
+    end
+
+    def edit
+      @company_value = CompanyValue.find(params[:id])
+    end
+
+    def new
+      @company_value = CompanyValue.new
+    end
+
+    def create
+      company_value_duplicate = CompanyValue.find_by(title: company_value_params[:title])
+
+      if company_value_duplicate.present?
+        redirect_to new_admins_company_value_url, notice: 'Title must be unique.'
+        # render json: {errors: {message: 'The same uuid already exists.'}}, status: :accepted
+        return
+      end
+
+      @company_value = CompanyValue.new(company_value_params)
+
+      if @company_value.title.empty?
+        redirect_to new_admins_company_value_url, notice: 'Title is empty. Company Value was not saved.'
+      else
+        # if params[:company_value][:title].exists?
+        #   redirect_to new_admins_company_value_url, notice: 'Title must be unique.'
+        # else
```

Comment:
comments generally make the code hard to read, I would get rid of them altogether